CHANGELOG.md

```diff
@@ -23,6 +24,167 @@
 - [0.10.0](#0100---20170307)
 - [0.9.9 and prior](#099---20170202)
 
+## [1.1.0]
+
+> Released on: 2019/03/27
+
+This is a minor release, introducing new features such as **Declarative
+Configuration**, **DB-less Mode**, **Bulk Database Import**, **Tags**, as well
+as **Transparent Proxying**. It contains a large number of other features and
+fixes, listed below. Also, the Plugin Development kit also saw a minor
+updated, bumped to version 1.1.
+
+This release includes database migrations. Please take a few minutes to read
+the [1.1 Upgrade Path](https://github.com/Kong/kong/blob/master/UPGRADE.md)
+for more details regarding changes and migrations before planning to upgrade
+your Kong cluster.
+
+### Additions
+
+##### Core
+
+- :fireworks: Kong can now run **without a database**, using in-memory
+  storage only. When running Kong in DB-less mode, entities are loaded via a
+  **declarative configuration** file, specified either through Kong's
+  configuration file, or uploaded via the Admin API.
+  [#4315](https://github.com/Kong/kong/pull/4315)
+- :fireworks: **Transparent proxying** - the `service` attribute on
+  Routes is now optional; a Route without an assigned Service will
+  proxy transparently
+  [#4286](https://github.com/Kong/kong/pull/4286)
+- Support for **tags** in entities
+  [#4275](https://github.com/Kong/kong/pull/4275)
+  - Every core entity now adds a `tags` field
+- New `protocols` field in the Plugin entity, allowing plugin instances
+  to be set for specific protocols only (`http`, `https`, `tcp` or `tls`).
+  [#4248](https://github.com/Kong/kong/pull/4248)
+  - It filters out plugins during execution according to their `protocols` field
+  - It throws an error when trying to associate a Plugin to a Route
+    which is not compatible, protocols-wise, or to a Service with no
+    compatible routes.
+
+##### Configuration
+
+- New option in `kong.conf`: `database=off` to start Kong without
+  a database
+- New option in `kong.conf`: `declarative_config=kong.yml` to
+  load a YAML file using Kong's new [declarative config
+  format](https://discuss.konghq.com/t/rfc-kong-native-declarative-config-format/2719)
+- New option in `kong.conf`: `pg_schema` to specify Postgres schema
```

Comment:
Some notes don't have a PR link, is it intentional?

Reply:
Yeah, many of them come from the same declarative config PR which was already linked. I didn't want to put the same PR number in n different places, but can do if it's the way we usually do it — is it?

Reply:
Usually yes, but we don't have to do it tbh

Reply:
Ok, I reviewed it for entries that were missing PRs (that were not already mentioned) and caught one and fixed it